client/src/components/application/Form.tsx

```diff
@@ -98,39 +86,49 @@ export default function Form({
   }, [dispatch]);
 
   useEffect(() => {
-    const currentOptions = props.projects.map((project) => ({
-      id: project.id,
-      title: props.allProjectMetadata[project.id].metadata?.title,
-    }));
+    const currentOptions = props.projects.map(
+      (project): ProjectOption => ({
+        id: project.id,
+        title: props.allProjectMetadata[project.id].metadata?.title,
+      })
+    );
+    currentOptions.unshift({ id: undefined, title: "" });
```

Comment:
I like how this doesn't default to a specific project, but maybe we should add validation so that you can't submit the application without a valid project

Reply:
yeah we have an error already if a project is not selected